backbone/src/main/java/org/researchstack/backbone/model/survey/NavigationStep.java

```diff
@@ -0,0 +1,24 @@
+package org.researchstack.backbone.model.survey;
+
+import org.researchstack.backbone.result.StepResult;
+import org.researchstack.backbone.result.TaskResult;
+import org.researchstack.backbone.step.QuestionStep;
+
+import java.util.List;
+
+/**
+ * Created by TheMDP on 12/31/16.
+ */
+
+public interface NavigationStep {
```

Comment:
The classes using NavigationStep I think just retrieve them.  And there is definitely more complex skip logic used by the NavigableOrderedTask in the iOS library; however, this is sufficient for now. 

 I'm thinking this NavigationStep interface will need iterated on when I start fully implementing the Steps in the on boarding process, and the UI they relate to.  I'll put a TODO in this class that it needs expanded to support SBANavigationRule, SBAConditionalRule, and SBANavigationSkipRule in the near-future.